docs/CLI-INSTALLATION.md

```diff
@@ -0,0 +1,67 @@
+macOS and GNU/Linux
```

Comment:
Can we make all the headings h2s, otherwise it's going to look the same size as the main headings on the webpage